src/dolphin/main.ts

```diff
@@ -47,10 +49,14 @@ removePlayKeyFile.main!.handle(async () => {
   return { success: true };
 });
 
-viewSlpReplay.main!.handle(async ({ filePath }) => {
+viewSlpReplay.main!.handle(async ({ filePath, startFrame }) => {
+  if (startFrame === undefined) {
+    startFrame = Frames.FIRST;
+  }
```

Comment:
I don't think we need this. `startFrame` can be left undefined according to the `ReplayCommunication` type as defined in `types`.ts.